### ISSUE TITLE
touch events become erratic when triple-tap to magnify android accessibility feature is enabled

### ISSUE THREAD

**Issue by Spartrap** [mozilla-mobile/fenix#16632]
This is on Android 10, firefox 82.1.3

Bug description:
when triple-tap to magnify accessibility feature is enabled, clicking links is erratic, works 1 time out of 3, tap to scroll often fails and the page doesn't scroll.
Disabling the accessibility feature fixes things.

Steps to reproduce:
-enable "triple-tap to magnify" accessibility feature of android
-open long html page with lots of links and try clicking and/or scrolling at various points.

**Comment by Spartrap** [comment]
I observe this on google search results page for instance. Try scrolling up and down a few times, then touching a search result. Also form field behavior  is erratic. It's difficult to put the focus inside. Just typing this message, moving the cursor around in the textarea is difficult.

**Comment by Spartrap** [comment]
Just tested on the nightly and it seems to work fine, so I'll close the report. Thanks.
